openapi/paths/aml-checks@{id}.yaml

```
@@ -0,0 +1,29 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve results of a Customer's AML check
+  operationId: GetAmlCheck
+  x-sdk-operation-name: get
+  description: |-
+    Retrieve the results of a Customer's AML check.
+    Customer metadata and an array of matching AML hits is returned.
+  parameters:
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+    - $ref: ../components/parameters/collectionSort.yaml
+  responses:
+    '200':
+      description: A list of AML checks was retrieved.
```

issue: where possible, avoid starting responses with `a` and using past tense `was`

also, I'm assuming that this operation returns one AML check for one customer

```suggestion
      description: Customers AML check retrieved.
```